pkg/koordlet/resmanager/resmanager.go

```diff
@@ -216,19 +223,19 @@ func NewResManager(cfg *Config, schema *apiruntime.Scheme, kubeClient clientset.
 // isFeatureDisabled returns whether the featuregate is disabled by nodeSLO config
 func isFeatureDisabled(nodeSLO *slov1alpha1.NodeSLO, feature featuregate.Feature) (bool, error) {
 	if nodeSLO == nil || nodeSLO.Spec == (slov1alpha1.NodeSLOSpec{}) {
-		return false, fmt.Errorf("cannot parse feature config for invalid nodeSLO %v", nodeSLO)
+		return true, fmt.Errorf("cannot parse feature config for invalid nodeSLO %v", nodeSLO)
 	}
 
 	spec := nodeSLO.Spec
 	switch feature {
 	case features.BECPUSuppress, features.BEMemoryEvict:
 		// nil value means enabled
-		if spec.ResourceUsedThresholdWithBE == nil {
-			return false, fmt.Errorf("cannot parse feature config for invalid nodeSLO %v", nodeSLO)
+		if spec.ResourceUsedThresholdWithBE == nil || spec.ResourceUsedThresholdWithBE.Enable == nil {
```

Comment:
these change the default setting of *BECPUSuppress* & *BEMemoryEvict* but i think it's acceptable. WDYT @zwzhang0107